core-skills/web-application-development-with-react/README.md

```diff
@@ -42,49 +48,60 @@ Learning the fundamentals - How not to hurt yourself with React
-- [ ] Has a make recipe for serve, which runs the application in development mode
-- [ ] Has a make recipe for test, which runs the tests for the application
-- [ ] Has a sandbox environment which allows them to quickly prototype components
-- [ ] Has the ablity to hot-reload based on changes 
-- [ ] Has a failing test that appears when running make test
+- Has created an application that is independent of the node install on the candidates machine 
+- Has a make recipe for serve, which runs the application in development mode
+- Has a make recipe for test, which runs the tests for the application
+- Has a sandbox environment which allows them to quickly prototype components
+- Has the ablity to hot-reload based on changes 
+- Has a failing test that appears when running make test
 
 #### Understanding
 
 The candidate can describe:
 
-- [ ] Situations in which using React would be a benefit
-- [ ] What a React component is
-- [ ] What are props within a component
-- [ ] What is state within a component
-- [ ] What children are within a component
-- [ ] How React allows you to respond to user events
-- [ ] One approach to styling react components
-- [ ] The component lifecyle
-- [ ] An approach to managing Application state
+- Situations in which using React would be a benefit
+- What a React component is
+- What are props within a component
+- What is state within a component
+- What children are within a component
+- How React allows you to respond to user events
+- One approach to styling react components
+- The component lifecyle
+- An approach to managing Application state
 
 ### Fox
 
-- Components
-  - The candidate has created components demonstrating the following:
-    - [ ] A component without any props or state
-    - [ ] The use of props
-    - [ ] The use of state
-    - [ ] The use of children
-    - [ ] Responding to user events
-    - [ ] Styling components  
-- Architecture
-  - The candidate can explain the following;
-    - [ ] Communicating with APIs
-    - [ ] Division of business logic, domain, rendering and state
-    - [ ] How to test component rendering
-    - [ ] How to test business logic
-
-**Coming Soon - Fox is intended to test ability to write simple components and understanding
-of architecture & testing strategies for complex applications**
+Getting crafty with React - Creating some components for your users
 
+- The candidate has created an application which contains:
+  - A component without any props or state
+  - A component which renders differently based on the following prop types being passed in:
+    - A boolean
+    - A string
+    - A function
+    - The `children` prop
+  - An appropriate use of the `key` prop
+  - A component which renders differently based on state changes
+  - A component which changes state in response to an onClick event
+  - A component which has been styled without global styles
```

Comment:
Could you clarify this? Does this mean not a library, or not using a css file?

Reply:
It means not using a single stylesheet for the entire application, which means either using a library or a stylesheet for the individual component. 

Just in a way that means you can take that component and move it into an isolated environment and it would still be styled

Reply:
Makes sense, I didn't immediately get that from the wording but that could be react inexperience.

Reply:
Given the level that we would expect someone to be taking this core skill, I think your concern is valid.

`A component which has styling that does not depend on a global stylesheet`?